src/Mapper/SoftDeletedMapper.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Yiisoft\Yii\Cycle\Mapper;
+
+use Cycle\Annotated\Annotation\Column;
+use Cycle\Annotated\Annotation\Table;
+use Cycle\ORM\Command\CommandInterface;
+use Cycle\ORM\Command\Database\Update;
+use Cycle\ORM\Context\ConsumerInterface;
+use Cycle\ORM\Heap\Node;
+use Cycle\ORM\Heap\State;
+use Cycle\ORM\Mapper\Mapper;
+
+/**
+ * @Table(
+ *      columns={"deleted_at": @Column(type="datetime", nullable=true)}
```

Comment:
Some additional documentation on usage would be very helpful.

Reply:
Added `@See` block and short comment